src/AWS.Deploy.Recipes/RecipeDefinitions/ASP.NETAppECSFargate.recipe

```diff
@@ -126,6 +129,7 @@
                     "Description": "The name of the new cluster to create.",
                     "Type": "String",
                     "DefaultValue": "{StackName}",
+                    "DisallowedValuesSource": "ECSClusterName",
```

Comment:
I really don't like the name `DisallowedValuesSource`. It seems like such a vague name. We could avoid adding yet another concept to recipes making them more complicated and just reuse `TypeHint`. The type hint here would be `NewECSClusterName`. Then we can put the logic in there.